src/app/app.module.ts

```diff
@@ -9,6 +9,12 @@ import { UsersPage } from '../pages/users/users';
 import { ReposPage } from '../pages/repos/repos';
 import { UserDetailsPage } from '../pages/user-details/user-details';
 import { UserLoginPage } from '../pages/user-login/user-login';
+import { I18nModule } from '../i18n/i18n.module';
```

Comment:
this module is from Angular?
import { TRANSLATIONS, TRANSLATIONS_FORMAT, LOCALE_ID } from '@angular/core';
https://angular.io/docs/ts/latest/cookbook/i18n.html